lib/tool-bar.js

```diff
@@ -20,10 +20,14 @@ async function useTouchBar() {
 }
 
 export function deactivate() {
-  toolBarView.destroy();
-  toolBarView = null;
-  touchBarManager.destroy();
-  touchBarManager = null;
+  if (toolBarView) {
+    toolBarView.destroy();
+    toolBarView = null;
+  }
+  if (touchBarManager) {
+    touchBarManager.destroy();
+    touchBarManager = null;
+  }
```

Comment:
Our babel config supports [optional chaining](https://github.com/suda/tool-bar/blob/b372ff7b296db6ca563fe5e781447d20ea30d9fd/babel.config.js#L9), so this nicer syntax would work:

```suggestion
  toolBarView?.destroy();
  toolBarView = null;
  touchBarManager?.destroy();
  touchBarManager = null;
```